crates/runtime/src/lib.rs

```diff
@@ -72,6 +81,27 @@ impl Runtime {
             },
         }
     }
+
+    pub fn start_pods_watcher(&mut self) -> notify::Result<()> {
+        let mut current_app = Arc::clone(&self.app);
+
+        let handle_event = move |event: podswatcher::PodsWatcherEvent| {
+            match event {
+                podswatcher::PodsWatcherEvent::PodsUpdated(new_app) => {
+                    tracing::debug!("updated pods information: {:?}", new_app);
```

Comment:
Unlike Go, it seems so far that the standard in Rust is to capitalize the first word in logs and errors.

Reply:
Fixed, thank you 👍